### ISSUE TITLE
Upload files error while submitting paddlecloud job 

### ISSUE THREAD

**Issue by wanghaoshuang** [PaddlePaddle/cloud#196]
submit.sh
```paddlecloud submit xxxxx ../my_package```

I put submit.sh into `my_package` directory. Uploading file job will be failed.
And i move  `submit.sh` to parent directory, files can be uploaded.

**Comment by Yancey1989** [comment]
Sorry, for the current design of `paddlecloud submit...`, the package path does not support parent directory, so you can submit the package with `./` instead of `../my_package`

**Comment by wanghaoshuang** [comment]
`paddlecloud submit` can`t upload directory structure. It seems that paddlecloud submit cmd just recursively walk a directory and put files to cloud. 
If my directory:
```
/dir_0
 |-dir_1
 |--file_0
 |--file_1
 |-dir_1
 |--file_2
 |--file_3
```
When i upload all the files, it will looks like ：
```
/dir_0
 |-file_0
 |-file_1
 |-file_2
 |-file_3

```

**Comment by Yancey1989** [comment]
Thanks for @wanghaoshuang , `paddlecloud` will support upload folders with this PR: https://github.com/PaddlePaddle/cloud/pull/193, and it's in Reviewing.

**Comment by Yancey1989** [comment]
This bug has already fixed by #193
